scheme/common/arm/handlers.go

```diff
@@ -60,7 +81,58 @@ func SynthKeysFromRefValue(scheme string, tenantID string,
 	log.Debugf("Scheme %s Plugin Reference Value Look Up Key= %s\n", scheme, lookupKey)
 
 	return []string{lookupKey}, nil
+}
+
+func GetReferenceIDs(
```

Comment:
Ditto.

Reply:
Similarly, there is no `GetReferenceIDs()` now only `GetPlatformReferenceIDs()` shared across all the Arm platforms